handler.fish

```diff
@@ -1,24 +1,14 @@
-function __fish_command_not_found_on_interactive --on-event fish_prompt
-    functions --erase __fish_command_not_found_handler
-    functions --erase __fish_command_not_found_setup
```

Comment:
This is [removed](https://github.com/fish-shell/fish-shell/commit/1a12071a48a4ead52c8749118aef2a1016bdc5e5#diff-0d2c4442bc9780ab3f6d9e4b371b7cd7c89c18aca56bb25204e94d244a8d683b) prior-to Fish 3.0 (more than 5 years ago), so definitely not needed anymore.